### ISSUE TITLE
Mission 4

### ISSUE THREAD

**Issue by johanjun** [johanjun/Google_Machine_Learning_Bootcamp_KR_2021#8]
### Course 2) Improving Deep Neural Networks: Hyperparameter tuning, Regularization and Optimization

- week 2) Optimization algorithms
Hyperparameter tuning
- week 3) Batch Normalization and Programming Frameworks

~9월 6일 오전 9시 까지
